nms/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js

```diff
@@ -226,6 +233,14 @@ export default function AddSubscriberButton(props: {onClose: () => void}) {
   );
 }
 
+export function isNwtypeEmpty(arr: core_network_type_restriction): boolean {
+  const l = arr.length;
+  if (l < 1) {
+    return true;
+  }
+  return false;
+}
+
```

Comment:
Remove this function - this can be reduced to a one line evaluation where this function is called.

Reply:
Did that.
`error={props.subscriberState.allowed_nw_types && props.subscriberState.allowed_nw_types.length < 1 ? true : false}`